x-pack/plugin/sql/qa/src/main/resources/math.csv-spec

```diff
@@ -99,31 +99,31 @@ SELECT MIN(salary) mi, MAX(salary) ma, YEAR(hire_date) year, ROUND(AVG(languages
 
       mi:i     |      ma:i     |     year:i    |ROUND(AVG(languages),1):d|TRUNCATE(AVG(languages),1):d|   COUNT(1):l    
 ---------------+---------------+---------------+-------------------------+----------------------------+---------------
-25324          |70011          |1987           |3.0                      |3.0                         |15             
-25945          |73578          |1988           |2.9                      |2.8                         |9              
-25976          |74970          |1989           |3.0                      |3.0                         |13             
-31120          |71165          |1990           |3.1                      |3.0                         |12             
-30404          |58715          |1993           |3.0                      |3.0                         |3              
-35742          |67492          |1994           |2.8                      |2.7                         |4              
-45656          |45656          |1996           |3.0                      |3.0                         |1           
+25324          |70011          |1986           |3.0                      |3.0                         |15             
```

Comment:
where is this `-1 year` change come from?

Reply:
Relative vs absolute time computing. ES allows both absolute offsets (24h) or relative (1d) which takes into account if the date/time is within a day.
Based on the intervals given, ES keeps going back and forth but that's surprising and unexpected - in SQL as the intervals vary, I've used the absolute one so that `INTERVAL 1 YEAR` and `INTERVAL 12 MONTHS`, etc... gives you the same results.